docs/source/example_api_asreview_file.ipynb

```diff
@@ -36,7 +36,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
```

Comment:
Please reset the kernel and rerun the notebook. This avoids non-linear execution errors and changes in git.